source/eitri/requirements.txt

```diff
@@ -2,3 +2,5 @@ docker-py==0.4.0
 retrying==1.2.3
 psycopg2==2.4.5
 clingon==0.1.4
+GeoAlchemy2==0.2.4
+SQLAlchemy==1.0.4
```

Comment:
Is the need for alembic covered by SQLAlchemy?
Nota: I only needed alembic, no SQLAlchemy